src/lib/Components/Bidding/Screens/BillingAddress.tsx

```diff
@@ -88,52 +89,56 @@ export class BillingAddress extends React.Component<BillingAddressProps, Billing
   render() {
     return (
       <BiddingThemeProvider>
-        <ScrollView>
-          <Container>
-            <Title mt={0} mb={6}>
-              Your billing address
-            </Title>
-
-            <StyledInput
-              label="Full name"
-              placeholder="Enter your full name"
-              autoCapitalize="words"
-              {...this.propsForInput("fullName")}
-            />
-
-            <StyledInput
-              label="Address line 1"
-              placeholder="Enter your street address"
-              autoCapitalize="words"
-              {...this.propsForInput("addressLine1")}
-            />
-
-            <StyledInput
-              label="Address line 2 (optional)"
-              placeholder="Enter your apt, floor, suite, etc."
-              autoCapitalize="words"
-              {...this.propsForInput("addressLine2")}
-            />
-
-            <StyledInput label="City" placeholder="Enter city" {...this.propsForInput("city")} />
-
-            <StyledInput
-              label="State, Province, or Region"
-              placeholder="Enter state, province, or region"
-              autoCapitalize="words"
-              {...this.propsForInput("state")}
-            />
-
-            <StyledInput
-              label="Postal code"
-              placeholder="Enter your postal code"
-              autoCapitalize="words"
-              {...this.propsForInput("postalCode")}
-            />
-
-            <Button text="Add billing address" onPress={() => this.onSubmit()} />
-          </Container>
-        </ScrollView>
+        <View>
+          <BackButton navigator={this.props.navigator} />
```

Comment:
This looks like a large change, but you change GitHub's "Diff settings" and hide whitespace changes it'll be a lot easier to review:

<img width="250" alt="screen shot 2018-07-02 at 5 41 15 pm" src="https://user-images.githubusercontent.com/386234/42187762-30234904-7e1f-11e8-8456-4eb138ecc11d.png">